<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginOn</class>
 <widget class="QWidget" name="LoginOn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="User">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Agency FB&quot;;
border-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Agency FB&quot;;
border-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="User_shuru">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Mima_Shuru">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="LogOn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Register_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-15</x>
     <y>1</y>
     <width>631</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/20.jpg);</string>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>User</zorder>
  <zorder>Password</zorder>
  <zorder>User_shuru</zorder>
  <zorder>Mima_Shuru</zorder>
  <zorder>LogOn</zorder>
  <zorder>Register_Button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
